### PR TITLE
Fixing some links and wording

### PR DESCRIPTION
**Pull request by baberlevi**
Trying to follow some of the guides today, and found it pretty frustrating. Trying to update as I figure things out

### CODE REVIEW

File: src/splash/metag/index.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 title: Latest Tutorials
 date: 2025-03-04
 ---
-Check out [the latest metagenomics tutorial](https://galaxyproject.github.io/training-material//Metagenomics/tutorials/mothur-miseq-sop) by [@bebatut](https://twitter.com/bebatut) and [@shiltemann](https://twitter.com/shiltemann)!
+Check out [the latest metagenomics tutorial](http://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html) by [@bebatut](https://twitter.com/bebatut) and [@shiltemann](https://twitter.com/shiltemann)!
```
Comment (bgruening) [inline src/splash/metag/index.md]: This link can actually be now replaced with: `training.galaxyproject.org/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html`

**Comment by bgruening** [comment]
So many nice updates, thanks a lot! I added one small comment.

**Comment by martenson** [comment]
@bgruening I replace that link directly in master

**Comment by martenson** [comment]
Thanks a lot @baberlevi !

**Pull request merged**